sunpy/image/transform.py

```diff
@@ -12,6 +12,8 @@
     import skimage.transform
     scikit_image_not_found = False
 except ImportError:  # pragma: no cover
+    warnings.warn("scikit-image could not be imported. Map rotation will use scipy",
```

Comment:
The warning shouldn't refer to `Map` in specific since this transformation could be used more generally.